plugins/woocommerce-blocks/tests/e2e/tests/store-notices/store-notices.block_theme.spec.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+
+const blockData = {
+	name: 'Store Notices',
+	slug: 'woocommerce/store-notices',
+};
+
+test.describe( `${ blockData.slug } Block`, () => {
+	test( "block can't be inserted in Post Editor", async ( {
```

Comment:
I'm not sure if that's a bug or intended, but the Store Notices block seems to be available in the post editor. :thinking:

Reply:
Mmm, interesting! I can confirm that I'm able to add the Store Notice block in the Post Editor. I refactored it with [94210f9](https://github.com/woocommerce/woocommerce/pull/46671/commits/94210f976717daab873bb1abf8b93580e7811439)